README.md

```diff
@@ -70,25 +70,32 @@ To build a simple vector store index:
 import os
 os.environ["OPENAI_API_KEY"] = 'YOUR_OPENAI_API_KEY'
 
-from llama_index import GPTSimpleVectorIndex, SimpleDirectoryReader
+from llama_index import GPTVectorStoreIndex, SimpleDirectoryReader
 documents = SimpleDirectoryReader('data').load_data()
-index = GPTSimpleVectorIndex.from_documents(documents)
+index = GPTVectorStoreIndex.from_documents(documents)
 ```
 
-To save to and load from disk:
-```python
-# save to disk
-index.save_to_disk('index.json')
-# load from disk
-index = GPTSimpleVectorIndex.load_from_disk('index.json')
-```
 
 To query:
 ```python
 query_engine = index.as_query_engine()
 query_engine.query("<question_text>?")
 ```
 
+
+By default, data is stored in-memory.
+To persist to disk (under `./storage`):
+
+```python
+index.storage_context.persist()
+```
+
+To reload from disk:
+```python
+index = load_index_from_storage()
```

Comment:
is this the right call to load the index from storage? with blank args? 

in general i find `load_index_from_storage` with blank args a bit weird, like what conceptually does that mean

Reply:
Good point, ya agreed it's a little strange. 

The behavior is using the default storage context (i.e. in memory docstore, vector store, and index store), and assume there's a single index stored. I think perhaps better to revise to enforce always passing in a storage context.